# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: jamessan@debian.org-20120514004029-t728qa5zxkjgeguh
# target_branch: nosmart+http+urllib://anonscm.debian.org/bzr/apt\
#   /debian-sid/
# testament_sha1: 34d54df6cb94d3eaca96057d7372dfc4efeab07e
# timestamp: 2012-05-13 20:56:07 -0400
# base_revision_id: egon@debian-devbox-20120511170814-8njwsbos6etgc407
# 
# Begin patch
=== modified file 'apt-pkg/contrib/strutl.cc'
--- apt-pkg/contrib/strutl.cc	2012-04-21 16:26:49 +0000
+++ apt-pkg/contrib/strutl.cc	2012-05-14 00:40:29 +0000
@@ -1518,7 +1518,6 @@
    U.User.clear();
    U.Password.clear();
    U.Path.clear();
-   U.Port = 0;
    return U;
 }
 									/*}}}*/
@@ -1530,7 +1529,6 @@
    ::URI U(URI);
    U.User.clear();
    U.Password.clear();
-   U.Port = 0;
    return U;
 }
 									/*}}}*/

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWXXMkyQAAXXfgAAQWWf//3o3
2gC////6UAN6WpTAPO07CSSam0TUzRoNGmhJtJsE1MamynqaM1HpPUGSJhGTSYSeU9JiDTQAAAAA
JIKaaJk0ptqHlI09EAA0AAaA0OYTAJgBMJhNMAAAmTTQMJSSZBpk0MTQNNAGgAAADQ7Wp8/aZbs2
N+2txSx0TvpFOR6adj5dWxmZfT4HXNGGbBnTszvnnI7hx9oorNrm3Tc4MHZdYxRak5kV/ByHFRPT
Y+V7i1LCF7sk2JYzZdi5okSnaFUuzujTMDjM62J6ZhEOLkcwFLJkErDFcCMkzJrVRdNYuvHvcpcr
F8BWNF7ZHZoiWukCGBJWCLunEwB9Svl+Y1G5Bx0AyXL3+ucWx01OJ4OWaF63CKiHUTUbRhzjoaFg
byVSmApscFbAhpuaygMwt+FiEXh6xBWupweEDSBXEHUVmkIvPKqbaegtgIC1oVcIGZT7AsUnMWkD
POja2VIKza94zY1OZLCMKNjL3WLbBlmSce8jbivGoZYlQ0boNc9jTRQbJ9y8B7rFmsVunqR0IjOX
gFyabr8uNQQRVkeuIyCVt+AwY/Io2lNccqQvYBUxHsR6wxSZ7PJQjg1WMM+svxVTd+GJjN56dOlQ
EU2lasgxx88z/cogqGW+xq5j1H1gb5zIq1iWxP8VgXD1Z4ZbqMUcQweBp6C0DtXVrdmziDtrXp6X
P38pUWBEqTKWRQrnvntOiC6TrfVsl3bxsRnuQpEXl1VF9d2xEm1eR2/2C+E/jc6xXjhmyClopdZC
i+UR2IiRPVx2jXGn3RE3JY6s5Dbd5GCOriYSw91wpyZjX3YvDCOdNANugRVc7TsrroabMphSzWoN
R4uKTvTLqruC19DLS7KOQclLImCCfRTTlK4mGkbITGurTHjcTFtK2tCNovriX8GFqC4zJYQsxRY3
CXPGrwgpJEoosqPaitUx4dgSUPjAN8YHZTwN2wdhXj5nQVOZaEZSqL4nw3iU3iJVgHAcWBc7KYeh
WkiK7uoPRwVpxMlsNKRJI6hkmQwhg7nPEhbY7LPN5LrBXkQZMpetmpD1OvVO3UKMxh04JTLyHrWq
L/RC5tc81pqzKSsbO3ZbqJqhaF4QuxQ6LredyUrbF0wr90YTRpZOyYfoAzWk/jG3g5fQGuPM1QMU
aBwJGbHNAHQMQybUHJKym4PkZ66ZC6IFICTKMX1zirhbkjUiYJrG5OWe+anCjT8ktDGHzv0Enp9x
YcH/F3JFOFCQdcyTJA==
